syncano/models/manager.py

```diff
@@ -173,7 +193,30 @@ def bulk_create(self, *objects):
         .. warning::
             This method is not meant to be used with large data sets.
         """
-        return [self.create(**o) for o in objects]
+
+        if len(objects) > 50:
+            raise SyncanoValueError('Only 50 objects can be created at once.')
+
+        class_name = objects[0].get('class_name')
+        for o in objects[1:]:
+            next_class_name = o.get('class_name')
+            if o.get('class_name') != class_name:
+                raise SyncanoValidationError('Bulk create can handle only objects of the same type.')
```

Comment:
Well. This is django way. In django the snipper will be: Book.objects.bulk_create([Book(title='one'), Book(title='two'), ...]); In django you cannot pass to the bulk_create other model than is specified on the begining. I think this restriction allows to write more clean code. 

But this is not a real issue here. A real issue here is that when list of objects is returned, the population is made - and providing a bulk create for different objects will (different classes) will be much more complicated. Of course not impossible, but probably tricky.